docs/CHANGELOG.asciidoc

```diff
@@ -35,6 +35,7 @@
 * Upgrade PyTorch to version 1.11. (See {ml-pull}2233[#2233], {ml-pull}2235[#2235]
   and {ml-pull}2238[#2238].)
 * Upgrade zlib to version 1.2.12 on Windows. (See {ml-pull}2253[#2253].)
+* Address root cause for actuals equals typical equals zero anomalies. (See {ml-pull}2270[#2270].)
 
 === Bug Fixes
```

Comment:
Since you're editing this file anyway you could move the release note for PR 2261 (line 48) into the 8.2.1 section.  We changed our minds about which version to release it in but only updated the release notes on the 8.2 branch.

Reply:
Actually don't worry about this - I've done it in #2272